config/database.php

```diff
@@ -55,7 +55,7 @@
             'collation' => env('DB_COLLATION', 'utf8mb4_unicode_ci'),
             'prefix' => '',
             'prefix_indexes' => true,
-            'strict' => true,
+            'strict' => false,
```

Comment:
@axnish27 Please explain why did you disable the strict mode in MySQL

Reply:
[MySql Error: 1364 Field 'display_name' doesn't have default value](https://stackoverflow.com/questions/18751291/mysql-error-1364-field-display-name-doesnt-have-default-value)
 
i was getting a similar issue then as solution according to this post i changed the mysql to run on unstrict mode which resolved the issue
 
 @Raza9798

Reply:
once you disable the `strict mode` it will continue the querying process like `create, update, delete` even the input are invalid. just given an idea what it may cause. @axnish27

Reply:
Yes i can understand....i did this in a desperation to fix the issue...now only i thought about the consequences ill keep it as true and will find an appropriate solution @Raza9798